components/playlist/playlist_service.cc

```diff
@@ -120,47 +120,88 @@ void PlaylistService::RequestDownloadMediaFilesFromPage(
   download_request_manager_->GetMediaFilesFromPage(std::move(request));
 }
 
-void PlaylistService::RemoveItemFromPlaylist(const std::string& playlist_id,
-                                             const std::string& item_id) {
-  VLOG(2) << __func__ << " " << playlist_id << " " << item_id;
+bool PlaylistService::AddItemsToPlaylist(
+    const std::string& playlist_id,
+    const std::vector<std::string>& item_ids) {
+  prefs::ScopedDictionaryPrefUpdate playlists_update(prefs_, kPlaylistsPref);
+  std::unique_ptr<prefs::DictionaryValueUpdate> target_playlist_update;
+  if (!playlists_update->GetDictionary(playlist_id, &target_playlist_update)) {
+    LOG(ERROR) << __func__ << " Playlist " << playlist_id << " not found";
+    return false;
+  }
+
+  base::Value::List* ids_list = nullptr;
+  if (!target_playlist_update->GetListWithoutPathExpansion(kPlaylistItemsKey,
+                                                           &ids_list)) {
+    NOTREACHED() << __func__ << " Playlist " << playlist_id
```

Comment:
How about creating items key with this new item instead of early return?
Or we should not recover in here?

Reply:
I think this code is somewhat 'unlikely to be reachable' code. So I feel like creating key would be much for this code. Actually, I feel like chaning this to 

```
base::Value::List* ids_list = nullptr;
target_playlist_update->GetListWithoutPathExpansion(kPlaylistItemsKey,
                                                           &ids_list);
DCHECK(ids_list);
```
 Then we can remove early return.